matching-core/src/main/java/com/graphhopper/matching/EdgeMatch.java

```diff
@@ -68,6 +76,38 @@ public double getMinDistance() {
         return min;
     }
 
+    /**
+     * For OSM a way is often a curve not just a straight line. These nodes are called pillar nodes
+     * and are between tower nodes (which are used for routing), they are necessary to have a more
+     * exact geometry. Updates to the returned list are not reflected in the graph, for that you've
+     * to use setWayGeometry.
+     * <p>
+     * @param mode can be <ul> <li>0 = only pillar nodes, no tower nodes</li> <li>1 = inclusive the
+     * base tower node only</li> <li>2 = inclusive the adjacent tower node only</li> <li>3 =
+     * inclusive the base and adjacent tower node</li> </ul>
+     * @return pillar nodes
+     */
+    public PointList getWayGeometry(int mode){
```

Comment:
here I would use the same naming 'fetchWayGeometry' as in the edge interface because it is also here not always a cheap 'get' (?)